templates/localgov-alert-banner.html.twig

```diff
@@ -0,0 +1,68 @@
+{#
+/**
+ * @file localgov_alert_banner.html.twig
+ * Default theme implementation to present Alert banner data.
+ *
+ * This template is used when viewing Alert banner pages.
+ *
+ *
+ * Available variables:
+ * - content: A list of content items. Use 'content' to print all content, or
+ * - attributes: HTML attributes for the container element.
+ * - display_title: Boolean indicating if alert banner title should be displayed
+ * - remove_hide_link: Boolean indicating if the link to close the banner should be hidden
+ * - type_of_alert: Type of alert banner
+ *
+ * @see template_preprocess_localgov_alert_banner()
+ *
+ * @ingroup themeable
+ */
+#}
+
+{{ attach_library('localgov_alert_banner/alert_banner') }}
+
+{% set has_link = content.link.0 is not empty %}
+{% set type_of_alert =  type_of_alert|split('--')[1]|clean_class %}
+{% set classes = [
+    'js-localgov-alert-banner',
+    'localgov-alert-banner',
+    type_of_alert ? 'localgov-alert-banner--' ~ type_of_alert,
+    is_front ? 'localgov-alert-banner--homepage',
+    has_link ? 'localgov-alert-banner--has-link' : 'localgov-alert-banner--no-link'
+  ]
+%}
+
+<article {{ attributes.addClass(classes) }} role="alert">
```

Comment:
Are we sure we want to use `<article>` here?
An alert banner isn't really an article, though it might point to one, or it could just be a general alert saying forms are down.

Reply:
An `<article>` in HTML is also not just an "article" like you would expect in a newspaper (it is that, and more). It's a "self-contained composition". Article tag is also used for our media items (and other things) in Drupal.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article

These alerts are self-contained. They can display on any page and do not lose their meaning depending on the page they are on. They also don't lose their meaning if they were standalone on a page with nothing else.

We can move it back to div if you wish.

Reply:
Ok, that makes sense. I take it we will do the same for all components of the site, so block will be getting the article markup too?

Reply:
It would depend on the block to be honest. Not all blocks should be article, and it might not make sense for maintainability to create a template for every block type just so we can set article as the element.

But in cases where we _do_ have a template, then it makes sense to make sure we have the most appropriate element chosen.